examples/nlp/md/tweet-classification-using-tfdf.md

```diff
@@ -347,26 +347,4582 @@ model_2.fit(train_ds)
, qini=None)), TrainLog(num_trees=151, evaluation=Evaluation(num_examples=None, accuracy=0.8283582329750061, loss=0.8095303773880005, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None)), TrainLog(num_trees=152, evaluation=Evaluation(num_examples=None, accuracy=0.825373113155365, loss=0.808938205242157, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None)), TrainLog(num_trees=153, evaluation=Evaluation(num_examples=None, accuracy=0.8283582329750061, loss=0.8088335394859314, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None)), TrainLog(num_trees=154, evaluation=Evaluation(num_examples=None, accuracy=0.8283582329750061, loss=0.8086864948272705, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None)), TrainLog(num_trees=155, evaluation=Evaluation(num_examples=None, accuracy=0.8283582329750061, loss=0.8081754446029663, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None)), TrainLog(num_trees=156, evaluation=Evaluation(num_examples=None, accuracy=0.8283582329750061, loss=0.808414876461029, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None)), TrainLog(num_trees=157, evaluation=Evaluation(num_examples=None, accuracy=0.8298507332801819, loss=0.8094819188117981, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None)), TrainLog(num_trees=158, evaluation=Evaluation(num_examples=None, accuracy=0.8298507332801819, loss=0.8095358610153198, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None)), TrainLog(num_trees=159, evaluation=Evaluation(num_examples=None, accuracy=0.8268656730651855, loss=0.8095360994338989, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None)), TrainLog(num_trees=160, evaluation=Evaluation(num_examples=None, accuracy=0.825373113155365, loss=0.8099076747894287, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None)), TrainLog(num_trees=161, evaluation=Evaluation(num_examples=None, accuracy=0.825373113155365, loss=0.8096389174461365, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None)), TrainLog(num_trees=162, evaluation=Evaluation(num_examples=None, accuracy=0.825373113155365, loss=0.8101357221603394, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None)), TrainLog(num_trees=163, evaluation=Evaluation(num_examples=None, accuracy=0.8268656730651855, loss=0.8095107078552246, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None)), TrainLog(num_trees=164, evaluation=Evaluation(num_examples=None, accuracy=0.8268656730651855, loss=0.8108780980110168, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None)), TrainLog(num_trees=165, evaluation=Evaluation(num_examples=None, accuracy=0.8268656730651855, loss=0.8121252059936523, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None)), TrainLog(num_trees=166, evaluation=Evaluation(num_examples=None, accuracy=0.8298507332801819, loss=0.8118909001350403, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None)), TrainLog(num_trees=167, evaluation=Evaluation(num_examples=None, accuracy=0.8313432931900024, loss=0.8114330768585205, rmse=None, ndcg=None, aucs=None, auuc=None, qini=None))]
```

Comment:
Doesn't really look like something that adds value for the reader. Should you really print it?